src/main/frontend/extensions/zotero/handler.cljs

```diff
@@ -65,31 +65,26 @@
    (go
      (let [{:keys [page-name properties abstract-note]} (extractor/extract item)]
 
-       (if (page-handler/page-exists? (str/lower-case page-name))
-         (editor-handler/api-insert-new-block!
```

Comment:
If a page already exists, the default behavior is to just append a new block at the end of the current page. 
The reason is that the properties might be changed in the Zotero world. Any idea on merging them when a page (zotero item) already existed?

Reply:
You are right. We need a solution to sync the item between zotero and logseq. I will think about it.